### DISCUSSION TITLE
analysis of cascade dynamics in email threads

### DISCUSSION THREAD

**Discussion by sbenthall** [datactive/bigbang#514] [Ideas]
A research question based on conversations with @wazaahhh

Taking threads as a unit of analysis, look at their size and properties as a subgraphs of the larger interaction graph, such as clustering coefficient (indicating a tight conversation between participants or a far-ranging one?).  Also, look at the effectiveness of different participants in triggering cascades.

For sizes of cascades, look at several metrics and test to see their values as a function of cascade size. Plot them. Regression analysis later?

Teasing apart the contributions of different actors and how their interactions change over time and as a function of who they are interacting with.

Granovetter, Stanford, and Leskovec as background theory.

**Comment by sbenthall** [comment]
See #65 for proposal for early work on thread analysis

**Comment by wazaahhh** [comment]
Great!

Charlotte told me that she was trying to build a work group with you, Dav
and others? Is it the same group you told me about last time ?

Shall we at some point meet altogether and share interests ?

Thomas
